player/misc.c

```diff
@@ -147,7 +147,12 @@ double get_track_seek_offset(struct MPContext *mpctx, struct track *track)
         if (track->type == STREAM_AUDIO)
             return -opts->audio_delay;
         if (track->type == STREAM_SUB)
-            return -opts->subs_rend->sub_delay;
+        {
+            for (int n = 0; n < num_ptracks[STREAM_SUB]; n++) {
```

Comment:
this should probably be capped so n doesn't go above 1

Reply:
This code was deliberately copied from other places that do the same thing. Capping at 1 here does nothing since these will still exist.

https://github.com/mpv-player/mpv/blob/master/player/command.c#L6863
https://github.com/mpv-player/mpv/blob/master/player/sub.c#L41
https://github.com/mpv-player/mpv/blob/master/player/sub.c#L137
https://github.com/mpv-player/mpv/blob/master/player/sub.c#L210